app/components/filters/user_attributes.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Filters
+  class UserAttributes < ViewComponent::Base
+    attr_accessor :filters
+
+    def initialize(filters:)
+      @filters = filters
+      super
```

Comment:
```suggestion
      super
      @filters = filters
```
Always call `super` **before** initialising instance variables